es-ds-docs/pages/molecules/verification-code.vue

```diff
@@ -0,0 +1,124 @@
+<script setup lang="ts">
+const { $prism } = useNuxtApp();
+const compCode = ref('');
+const docCode = ref('');
+
+if ($prism) {
+    /* eslint-disable import/no-webpack-loader-syntax, import/no-self-import */
+    const compSource = await import('@energysage/es-ds-components/components/es-verification-code.vue?raw');
+    const docSource = await import('./verification-code.vue?raw');
+    /* eslint-enable import/no-webpack-loader-syntax, import/no-self-import */
+
+    compCode.value = $prism.normalizeCode(compSource.default);
+    docCode.value = $prism.normalizeCode(docSource.default);
+    $prism.highlight();
+}
+
+const propTableRows = [
+    ['v-model', 'String', 'n/a', 'Required. The v-model directive binds the otpinput to a data property.'],
+];
+const code = ref();
+const isSubmitInProgress = ref(false);
+
+const charCount: number = 5;
+
+const isValid = () => {
+    if (code.value && code.value.length === charCount) {
+        return true;
+    }
+    return false;
+};
+
+const randomCode = () => {
+    // Randomly generates an array of numbers 0-9
+    const rand = Array.from({ length: charCount }, () => Math.floor(Math.random() * 10).toString());
+
+    // eslint-disable-next-line no-console
+    console.log('Generated', rand);
+    isSubmitInProgress.value = true;
+    setTimeout(() => {
+        isSubmitInProgress.value = false;
+    }, 1000);
+    return rand.join('');
+};
+
+// eslint-disable-next-line no-return-assign
+const clearCode = () => {
+    isSubmitInProgress.value = true;
+    setTimeout(() => {
+        isSubmitInProgress.value = false;
+    }, 1000);
+    return '';
+};
+</script>
+
+<template>
+    <div>
+        <h1>Verification code</h1>
+        <p>{{ code }}</p>
+        <p class="mb-500">
+            Uses
+            <a
+                href="https://v3.primevue.org/inputotp/"
+                target="_blank">
+                PrimeVue InputOtp
+            </a>
+        </p>
+        <b-row class="justify-content-center my-500">
+            <es-verification-code
+                v-model="code"
+                :char-count="charCount"
+                :disabled="isSubmitInProgress" />
+        </b-row>
+        <b-row class="mt-200 align-items-center">
+            <b-col
+                class="mb-100 mb-lg-0"
+                cols="10"
+                lg="4">
```

Comment:
Due to our new design system docs pages more closely adhering to our content container max widths (unlike the old ones), this section is a bit width-constrained on most breakpoints and wraps awkwardly.

<img width="872" alt="Screen Shot 2024-10-02 at 5 14 31 PM" src="https://github.com/user-attachments/assets/1ba7fbdf-ae1a-4973-b2ec-4db99f31fb77">

Can you finesse the columns here at various breakpoints so the buttons either wrap to the next line fully underneath the text, or have enough room to go side-by-side with the text without one button wrapping underneath the other?